libs/core/src/lib/checkbox/checkbox/checkbox.component.ts

```diff
@@ -72,6 +72,10 @@ export class CheckboxComponent implements ControlValueAccessor {
     @Input()
     tristateSelectable = true;
 
+    /** Assigns given class to checkbox label element */
+    @Input()
+    labelClass: string;
```

Comment:
I think this should be hidden

Reply:
I think this is very useful input. Users often apply their classes to form label/values to customize their look. In this case label element is barely accessible.